perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1175,8 +1176,10 @@ sub garden_setup : Chained('property') : PathPart('') : CaptureArgs(0) {
 
     $c->detach('property_redirect') if $c->stash->{waste_features}->{garden_disabled};
 
-    $c->stash->{per_new_bin_cost} = $c->cobrand->_get_cost('ggw_new_bin_cost');
-    $c->stash->{per_new_bin_first_cost} = $c->cobrand->_get_cost('ggw_new_bin_first_cost') || $c->stash->{per_new_bin_cost};
+    $c->stash->{garden_costs} = WasteWorks::Costs->new({
+        cobrand => $c->cobrand,
+        service => $c->cobrand->garden_current_subscription,
```

Comment:
Not sure if `service` would be tidier being set up inside the package as already available from setting the cobrand - or used directly inside from the cobrand attribute without having an actual `service` attribute.

Reply:
Yeah, I'm not sure how this will go with future changes for new backend - it might actually be better to pass in the end date directly as that's all that's being used, but as you say, for now it might as well just get it from the cobrand and we can change again in future if need be.